buildall.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+# Used for PR builds, build a variety of things
+export MACHINE=x64
+bitbake packagegroup-ni-coreimagerepo
```

Comment:
Is the near-term goal here to build everything(x64) NIOE builds today and have os-common build on top of it? Or are you looking farther out?

Reply:
My goal is to only use this for PR builds at the moment, since they are difficult to do in the other pipeline (that will do the official exports).  Eventually this might be the thing that really does the build, but it's low overhead to get this working.

Reply:
If the intention is to check PRs in the fastest, most informative way, then I would recommend just bitbaking the x64 image targets and the desirables packagegroup.

```bash
bitbake nilrt-dkms-image
bitbake packagegroup-ni-desirable
```

That should be enough to trigger bitbake to build nearly all of what goes into the core feeds, and all of the extras packages which we guarantee.

The targets you have identified here are from the nilrt-nxg pipeline, btw. We don't build the coreimagerepo packagegroup and the restore-mode-image in the OneRT pipeline.

Reply:
Fixed in latest version.